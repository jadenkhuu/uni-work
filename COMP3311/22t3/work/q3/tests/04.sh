psql racing -c "select * from q3('Plate') order by race;"
