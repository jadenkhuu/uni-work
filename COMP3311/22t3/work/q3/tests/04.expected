              race              |         horses          
--------------------------------+-------------------------
 Breeders' Plate                | F2,F3,F3,M3,M5
 Craven Plate                   | F4,F4,F5,G3,M3,M4,M5,S4
 Frank Packer Plate             | F3,F3,G3,G3,M3,M4
 Japan Racing Association Plate | F3,F3,F4,F4,F4,M2,M3,M3
 Maribyrnong Plate              | F4,F4,G3,M3,M4,M4
 Oakleigh Plate                 | F3,F4,F4,G3,M3,M4
 Rough Habit Plate              | F3,F3,F3,F4,G3,M3,M4,M4
 Vo Rogue Plate                 | F3,F3,F4,F5,G4,M2,M4,S5
 W. S. Cox Plate                | F2,F4,G3,M4,M4,M4,M5,S4
(9 rows)
